apps/darts-modernisation/darts-api/stg.yaml

```diff
@@ -10,5 +10,6 @@ spec:
       ingressHost: darts-api.staging.platform.hmcts.net
       environment:
         DARTS_GATEWAY_URL: https://darts-gateway.staging.platform.hmcts.net
+        DARTS_LOG_LEVEL: DEBUG
```

Comment:
I take it this var is read by the application?

Reply:
yes added it in this pr https://github.com/hmcts/darts-api/pull/447